.github/workflows/build_and_test_workflow.yml

```diff
@@ -186,7 +186,7 @@ jobs:
         working-directory: ./artifacts
     strategy:
       matrix:
-        version: [ osd-2.0.0, osd-2.1.0, osd-2.2.0 ]
+        version: [ osd-2.0.0, osd-2.1.0, osd-2.2.0, osd-2.3.0 ]
```

Comment:
Do we need to include 1.3.2? https://github.com/opensearch-project/OpenSearch-Dashboards/pull/2284

Reply:
This change is going into main (v2.4.0) for the future changes to be backward compatible with 2.3.0. I see in #2284 1.3.2 was restored into 2.x branch. Do you mean we need to restore 1.3.2 into main?

Reply:
Good question yall. For `main`, it is on version 3.0.0. Via BWC tests we were actually able to realize it is not compatible to migrate directly from `1.x` to `3.x` which is set by OpenSearch reading which version our indices are from and rejecting the migration. So this helps us keep track and ping doc owners to ensure to note this.

tl;dr: 1.3.2 can't be added to main. Users will have to migrate from 1.x to 2.x to 3.x